client/src/app/menu/menu.component.ts

```diff
@@ -46,13 +39,8 @@ export class MenuComponent implements OnDestroy, OnInit {
         });
     }
 
-    openUserInformation(): void {
-        const dialogRef = this.dialog.open(ConnectedUserDialogComponent, {
-            position: {
-                top: '60px',
-                right: '20px'
-            }
-        });
+    isGuest(): Boolean {
+        return this.user.accountType === 'Guest' ? false : true;
```

Comment:
I was thinking to directly put it in the class `ConnectedUser` as we will probably use it from somewhere else